samples/README.md

```diff
@@ -75,8 +75,12 @@ For better readability and integrations with the sample test infrastructure, sam
 * The sample file should be either `*.py` or `*.ipynb`, and its file name is consistent with its directory name.
 * For `*.py` sample, it's recommended to have a main invoking `kfp.compiler.Compiler().compile()` to compile the 
 pipeline function into pipeline yaml spec.
-* For `*.ipynb` sample, parameters (e.g., experiment name and project name) should be defined in a dedicated cell and 
-tagged as parameter. Detailed guideline is [here](https://github.com/nteract/papermill). Also, all the environment setup and 
+* For `*.ipynb` sample, parameters (e.g., experiment name and project name)
+should be defined in a dedicated cell and tagged as parameter. 
+(If the author would like to let the sample test infra run it, it's necessary to parameterize 
+experiment name and project name used.)
```

Comment:
Curious where project name gets used in the test infra?

Reply:
Good catch. I didn't think carefully, it does not need to be a common setup.
Maybe in tfx-oss sample we'll need to do that? Removed for now.